### DISCUSSION TITLE
autopilot no longer works during altitude climb

### DISCUSSION THREAD

**Discussion by Drakkenblade** [flybywiresim/aircraft#3145] [Help]
I use the last development version of the A320.
I encounter a problem when climbing the airbus. Suddenly the autopilot no longer works and the airplane turns on itself. 
Autopilot remains activated but the aircraft no longer responds. 
I thought this issue had been resolved but I am encountering it again.
I have adjusted the settings of my yoke but nothing helps
Someone can help me ?

PS :I just realized that losing the autopilot happens when I touch the FMC. The Autopilot light is still on but it no longer works

**Comment by riveroalbert** [comment]
I have experiencied similar issues. AP stays on but plane does not follow the flight director. Specially critical when flying to direct waypoints.  I have also noticed the plane climbing and cruising sideways (at an angle) with no apparent reason (even with wind settings off) 
I’m sure it will be fixed soon

**Comment by MapleNAV-TCASFAULT** [comment]
What are your dead zone settings? start with 20% and see if it helps

**Comment by Drakkenblade** [comment]
Hello,
my dead zone for my Yoke is 10 % ( except for the mouse because i cant move th dead zone)

Edit : 
I just tried with the 20% dead zone and I have the same problem..
I have to deactivate the flight director and the autopilot then reactivate them to make it work again
